datus/storage/document/doc_init.py

```
@@ -0,0 +1,76 @@
+            if len(batch_data) == 24:
+                _save_batch(store, batch_data)
+                batch_data.clear()
+
+        if batch_data:
+            _save_batch(store, batch_data)
+
+        logger.info(f"Imported {imported_count} documents")
+        store.create_indices()
+        return imported_count, imported_titles
+
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)